Anatoly Shlom/3.js

```diff
@@ -0,0 +1,20 @@
+/*3) Complete the method/function so that it converts dash/underscore delimited words into camel casing. The first word within the output should be capitalized only if the original word was capitalized.
+
+Example:
+
+toCamelCase("the-stealth-warrior") // returns "theStealthWarrior"
+toCamelCase("The_Stealth_Warrior") // returns "TheStealthWarrior" */
+
+function toCamelCase(str) {
+    target=str.split('');
+    target.reduce(function(acc, cur, n, arr) {
+        if(cur == '-' || cur == '_') {
+            arr[n+1] = arr[n+1].toUpperCase();
+            return acc;
+        }
+        return acc += cur;
+    })
+    return target.join('');
+}
+console.log(toCamelCase("the-stealth-warrior"));
```

Comment:
please try to run this file in Node.js. I get this output
![image](https://user-images.githubusercontent.com/31970322/62608648-dff89d00-b908-11e9-8f2c-f03c7ce2cdd3.png)

Reply:
the same in the browser 
![image](https://user-images.githubusercontent.com/31970322/62609302-2d293e80-b90a-11e9-8c3a-df2b21cd100d.png)

Reply:
Fixed. Forgot about `arr.splice(n,1);`